### PR TITLE
feat: refactor all settings layouts

### PR DESCRIPTION
**Pull request by ansmonjol**
We wanted to refactor the settings pages in the app.

Goal was to also pass components to tailwind as much as possible.

In addition, there is more Tailwind config added:
- New spacing, classes and zIndex usable with Tailwind
- New layout components, which is probably the new standard for those kind of approaches.

Better review commit by commit

Fixes LAGO-372

### CODE REVIEW

File: cypress/e2e/t20-invitations.cy.ts [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+  //   cy.visit('/settings/members')
+  //   cy.get(`[role="dialog"]`).should('not.exist')
+  //   cy.get(`[data-test="create-invite-button"]`).click()
+  //   cy.get(`[role="dialog"]`).should('exist')
+  //   cy.get('[data-test="submit-invite-button"]').should('be.disabled')
+  //   cy.get('input[name="email"]').type(inviteEmail)
+  //   cy.get('[data-test="submit-invite-button"]').click()
+  //   cy.get('[data-test="copy-invite-link-button"]').click()
+  //   cy.get(`[role="dialog"]`).should('not.exist')
+  // })
```
[the remark this answers is not in the document]
Comment (ansmonjol) [reply cypress/e2e/t20-invitations.cy.ts]: Oupsi, was for test. Removed

File: src/components/layouts/Settings.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+import { tw } from '~/styles/utils'
+
+import { Skeleton, Typography } from '../designSystem'
+
+export const SettingsPaddedContainer = ({
+  children,
+  className,
+}: PropsWithChildren & { className?: string }) => (
+  <div className={tw('flex max-w-192 flex-col gap-10 px-4 pb-20 pt-8 md:px-12', className)}>
```
[the remark this answers is not in the document]
Comment (ansmonjol) [reply src/components/layouts/Settings.tsx:11]: Indeed! Main problem is the current container breakpoint does not correspond to our existing breakpoint.

I think it's a mistake and will probably fix and use it there.

Thanks!

[the remark this answers is not in the document]
Comment (ansmonjol) [reply src/components/layouts/Settings.tsx]: Fixed! We passed all to 776 and container should be spread over the app, and later we'll update all at once to implement new layout system

[the remark this answers is not in the document]
Comment (ansmonjol) [reply src/components/layouts/Settings.tsx]: Thanks for pointing that

**Review by keellyp — APPROVED** [review]

**Pull request merged**